test_timezone_integration.py

```
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+"""
+时区集成测试脚本
+验证 prompts API 的时区处理功能
+"""
+
+import requests
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)